engine/apps/social_auth/live_setting_django_strategy.py

```diff
@@ -2,7 +2,7 @@
 
 from django.conf import settings
 from django.shortcuts import resolve_url
-from django.utils.encoding import force_text
+from django.utils.encoding import force_str
```

Comment:
https://docs.djangoproject.com/en/4.2/internals/deprecation/#deprecation-removed-in-4-0